text/0000-ternary.md

```diff
@@ -0,0 +1,238 @@
++ Feature name: (fill me in with a unique ident, `my_awesome_feature`)
++ Start date: (fill me in with today's date, YYYY-MM-DD)
++ RFC PR: [iotaledger/bee-rfcs#0000](https://github.com/iotaledger/bee-rfcs/pull/0000)
++ Bee issue: [iotaledger/bee#0000](https://github.com/iotaledger/bee/issues/0000)
+
+# Summary
+
+In this RFC we introduce both the API and implementation of the `bee-ternary`
+crate, a general-purpose ternary manipulation crate written in Rust that may be
+used to handle ternary data in the IOTA ecosystem.
+
+# Motivation
+
+Ternary is a fundamental part of the IOTA ecosystem and is used throughout, even
+for fundamental features like wallet addresses and transaction identification.
```

Comment:
"...of the IOTA technology"

Reply:
That statement is true, but I wonder whether we want to emphasize on the "fundamental" part. Can we even write something like: "Ternary **has been** a fundamental part of the IOTA technology and is still used throughout today. This includes address generation, hashing, and signature verification, as well as internal transaction representation and proof-of-work."
Then maybe add something like a warning/disclaimer, maybe something like:
"Note, that the IOTA Foundation is attempting to reduce the ternary aspects of the protocol to better accommodate  for todays binary architectures [see](https://blog.iota.org/chrysalis-b9906ec9d2de), hence, this crate will be needed until a full transition to a new signing scheme."